docs/content/troubleshooting.md

```diff
@@ -1,113 +1,119 @@
-[//]: # (Doc of docs reference 23)
 [//]: # (TODO - tech writer pass)
 
 # Troubleshooting
 
-* [Errors](#errors)
-* [Exceptions](#exceptions)
-* [Problems](#problems)
-
-We also maintain a [known issues]({{urlRoot}}/known-issues) page for any open bugs in the SpatialOS GDK for Unity.
+<%(TOC)%>
 
+<%(Callout type="tip" message="We also maintain a [known issues]({{urlRoot}}/known-issues) page for any open bugs in the SpatialOS GDK for Unity.")%>
 
 ## Errors
 
-#### Error reported
-When building workers (Unity Editor menu: **Build For Cloud** > **my_worker_name**):
+#### Error building player because build target was unsupported
+When building workers by selecting **Build For Cloud** > **my_worker_name** the following error may occur:
 **_Error building player because build target was unsupported_**</br>
 
-**Issue and workarounds**<br/>
- You don't have the right Unity build settings enabled.<br/>
+**Cause**<br/>
+ You don't have the correct Unity build settings set.<br/>
+
+**Fix**<br/>
  In your Unity Editor:
 
-* Check what build support you have enabled (menu: **File** > **Build Settings**) and make sure you have your target build platform enabled. <br/>
+* Select **File** > **Build Settings** and make sure you have your target build platform enabled. <br/>
```

Comment:
I thought we had a diagnostic tool for this? Under the `SpatialOS` menu?

Reply:
yeah access it at `SpatialOS > Check build support`

Reply:
Tested this new diagnostic tool and documented. I've also added info regarding iOS and Android build support, as this tool errors when these are missing.